homeassistant/components/androidtv/const.py

```diff
@@ -0,0 +1,31 @@
+"""Android TV component constants."""
+DOMAIN = "androidtv"
+
+ANDROID_DEV = DOMAIN
```

Comment:
Why do you have two constants that are the same? Do they have different meanings, even though their values are the same?

Reply:
Yes, I use `ANDROID_DEV` to store the device created in `hass` data and retrieve it during media player entity initialization.
This is what I do:

```
    hass.data.setdefault(DOMAIN, {})[entry.entry_id] = {
        ANDROID_DEV: aftv,
        ANDROID_DEV_OPT: entry.options.copy(),
        "conf_listener": entry.add_update_listener(update_listener),
        "stop_listener": stop_listener,
    }
```

So they mean 2 different things, but the value can be the same.